cli/README.md

```diff
@@ -0,0 +1,377 @@
+
+### Creating a minor release
+
+Let's say you found a bug affecting all the users of your block. And, you want to fix it on the fly to all your block users.
+
+First, make your changes and build:
+
+```
+npm run build
+```
+
+Then, run the update command:
+
+```
+element update
+```
+
+> Pro Tip: Don't worry, this will create a new staging version and won't be shipped just yet.
+
+Go to your Site Designer instance and test the change. As you are the block author, the Site Designer will show you the staging version of the block for your tests.
+
+When you are ready, just create a new release:
+
+```
+element release -n "Bug fix notes"
+```
+
+> Pro Tip: It's time to create another commit: `git commit -m "v1 bug fix released"`
+
+At this point, after a page refresh, all the users of your block will get the updates. They don't have to do anything.
+
+### Rolling back a minor release
+
+We are humans. What if we made a mistake and released something that was not ready yet?
+
+We can use the `rollback` command.
+
+The `rollback` command will jump back between minor versions, until hitting the major version. So, you can rollback until reaching the major release of the version you are currently at.
+
+In our example, we had a release for the major version, that we tagged with the commit message `v1 released`, and we also released a minor version with the commit message `v1 bug fix released`. Assuming tha the bug fix went terribly wrong, we can do:
+
+```bash
+element rollback
+```
+
+The system will mark the bug fix as staging again, so you can do proper fixing before releasing again. What this means, is that users will get the released version of the block, in our example, the major release tagged with `v1 released`.
+
+You can then fix the issue and release again:
+
+```
+npm run build
+element release -n "v1 bug fix second try"
+```
+
+> Pro Tip: It's a good time to create a new commit, as a checkpoint of the released block
+
+Or, maybe, you don't want to release again, but discard the staging version of the block and start fresh. You can do that by running the rollback command one more time:
+
+```
+element rollback
+```
+
+This will remove the staging version of the block, and ship the initial version to the users.
+
+> Pro Tip: We recommend to checkout your repo and point your code to the commit `v1 released`
+
+You can then make changes, and start fresh, from the `v1 released` commit state.
+
+### Creating a Major release
+
+Maybe you are adding new features to your block, the block configuration is not compatible with the previous, and/or the look and feel of the block changed. Those are all good reasons to create a major release.
+
+In order to do that, we recommend to create a new branch, where you will keep the `version 2` of the code:
+
+```bash
+git checkout -b version2
+```
+
+Then, run the following `publish` command with the `-m` flag:
```

Comment:
What does the "-m" flag do by itself? Or should there be a commit message?

Reply:
The -m is not related to git. Is just `m` from `major`